docker-compose.travis.yml

```diff
@@ -0,0 +1,44 @@
+version: '3'
+
+services:
+  mdb:
+    image: mongo:3.4.3
+    ports:
+      - "27017"
+    command: mongod
+  bdb:
+    depends_on:
+      - mdb
+    build:
+      context: .
+      dockerfile: ./compose/travis/Dockerfile
+      args:
+        backend: localmongodb
+    volumes:
+      - .:/usr/src/app/
+    environment:
+      BIGCHAINDB_DATABASE_BACKEND: localmongodb
+      BIGCHAINDB_DATABASE_HOST: mdb
+      BIGCHAINDB_DATABASE_PORT: 27017
+      BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
+      BIGCHAINDB_WSSERVER_HOST: 0.0.0.0
+      BIGCHAINDB_START_TENDERMINT: 0
+      TENDERMINT_HOST: tendermint
+      TENDERMINT_PORT: 46657
```

Comment:
Would you mind to use `BIGCHAINDB_TENDERMINT_HOST` and `BIGCHAINDB_TENDERMINT_PORT`? In this way the [`env_config` function](https://github.com/bigchaindb/bigchaindb/blob/e377fb57af609c491e9faaa27ecf8953139ce685/bigchaindb/config_utils.py#L116) can pick it up easily.

Reply:
Oh sure. But we should address this in a different PR as some code in multiple places will need to be changed.

Reply:
OK, we can have it right after this one.